src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static FileReaderImpl fileReader;
+    private static final String EMPTY_PATH = "";
+    private static final String CORRECT_PATH = "src/test/resources/filesFruitShop.csv";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void read_emptyPath_Ok() {
```

Comment:
```suggestion
    public void read_emptyPath_NotOk() {
```

it's negative test, lets use NotOk in test naming(check another tests)